app/components/javascript_required_component.html.erb

```diff
@@ -0,0 +1,28 @@
+<noscript>
+  <%= render StatusPageComponent.new(status: :error) do |c| %>
+    <% c.header { header } %>
+
+    <% if intro %>
+      <p><%= intro %></p>
+    <% end %>
+
+    <p><%= t('components.javascript_required.browser_instructions') %></p>
+
+    <ul class="usa-list">
+      <% browser_resources.each do |resource| %>
+        <li><%= new_window_link_to resource[:name], resource[:url] %></li>
+      <% end %>
+    </ul>
+
+    <p><strong><%= t('components.javascript_required.next_step') %></strong></p>
+  <% end %>
+  <link rel="stylesheet" href="<%= no_js_css_path %>">
```

Comment:
> will this load synchronously? are there any race condition cases we need to worry about?

It does load synchronously, which is also why I put it toward the bottom of the content, to avoid blocking other content.

I suppose it's theoretically possible to refresh the page before the stylesheet loads, though pretty improbable, and even then, the alert itself isn't critical content imo.